js/accordionModel.js

```diff
@@ -0,0 +1,74 @@
+define([
+    'core/js/models/itemsModel'
+], function(ItemsModel) {
+
+    var AccordionModel = ItemsModel.extend({
+
+        defaults: function() {
+            return _.extend({
+                _shouldCollapseItems: true,
+                _toggleSpeed: 200,
+                _activeItems: []
+            }, ItemsModel.prototype.defaults);
+        },
+
+        initialize: function() {
+            if (this.get('_shouldCollapseItems') === false) {
+                this.set('_activeItems', []);
+            }
+
+            _.each(this.get('_items'), function(item) {
+                item._isActive = false;
+            });
+
+            ItemsModel.prototype.initialize.apply(this, arguments);
+        },
+
+        checkCompletionStatus: function() {
```

Comment:
This function should be on ```ItemsModel``` as a getter. The if statement is common to all three ```ItemsModel``` components.

```
getCompletionStatus: function() {
    return (this.getVisitedItems().length == this.get('_items').length);
}
```